creating_images/guidelines.adoc

```diff
@@ -299,45 +299,34 @@ privileged user.
 Because the user ID of the container is generated dynamically, it will not have
 an associated entry in *_/etc/passwd_*. This can cause problems for applications
 that expect to be able to look up their user ID. One way to address this problem
-is to use link:https://cwrap.org/nss_wrapper.html[nss wrapper] and dynamically
-create a *_passwd_* file with the container's user ID as part of the image's
-start script:
+is to dynamically create a *_passwd_* file entry with the container's user ID as part
+of the image's start script. This is what a Dockerfile might include:
 
 ----
-export USER_ID=$(id -u)
-export GROUP_ID=$(id -g)
-envsubst < ${HOME}/passwd.template > /tmp/passwd
-export LD_PRELOAD=/usr/lib64/libnss_wrapper.so
-export NSS_WRAPPER_PASSWD=/tmp/passwd
-export NSS_WRAPPER_GROUP=/etc/group
+RUN chgrp -R 0 /some/directory \
+    && chmod -R g=u /some/directory /etc/passwd
```

Comment:
> In regards to the -R, that is required for '/some/directory'. 

hmm, right, forgot you're doing both in that line.  

> Begs the question though, do we repeat the /some/directory chgrp and chmod since its also shown above? or do we limit this docs section to the single file chmod g=u /etc/passwd?

good point, i would vote for not repeating it.  this part of the docs is just about how to get a random uid into /etc/passwd so "whoami" works, not about ensuring that random uid has permission to access particular files/folders (which is what the previous bit covers)